src/RunTests/RunTestsTask.cs

```diff
@@ -0,0 +1,443 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+//
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Build
+{
+    public class RunTestsTask : Build.Utilities.Task
```

Comment:
I would rename in `RunVSTestTask`

Reply:
I agree with this suggestion, the whole task and target should use VSTest in the name.

Reply:
Updated the name of the task and target to use VSTest